packages/core/strapi/src/commands/index.ts

```diff
@@ -114,14 +119,14 @@ const buildStrapiCommand = async (argv: string[], command = new Command()) => {
   } satisfies CLIContext;
 
   // Load all commands
-  keys.forEach((name) => {
+  for (const name of keys) {
     try {
       // Add this command to the Commander command object
-      strapiCommands[name]({ command, argv, ctx });
+      await strapiCommands[name]({ command, argv, ctx });
     } catch (e) {
       console.error(`Failed to load command ${name}`, e);
     }
-  });
+  }
```

Comment:
Do we need building the commands to be async? I think they're all synchronous. But if it still works, I guess it doesn't really matter.

Reply:
StrapiCloudCommands is an async func as we need to use FS, and Alex asked to turn it into async, to generate the deviceId and storing it in user's file system